input/gen_equilibrium_table.py

```diff
@@ -60,22 +63,24 @@
 # Call convenience function for setting up a fluid container.
 # This container holds the solver parameters, units, and fields.
 fc = setup_fluid_container(chem,
-                           density=d.flatten(),
-                           temperature=t.flatten(),
-                           metal_mass_fraction=0.02014*z,
-                           converge=True)
+                           density=d.ravel(),
+                           temperature=t.ravel(),
+                           metal_mass_fraction=0.01295*Z,
```

Comment:
This is the solar mass fraction used by both Grackle & Enzo because of the CLOUDY version used for the Grackle tables (see grackle_chemistry_data_fields.def:111 for Grackle & SetDefaultGlobalValues.C:524 for Enzo). I think originally I had used a value off of Wikipedia before finding it in the codebase, so I changed it for internal consistency.

Reply:
That makes sense.  Thank you!